app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,6 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
+ *= require_tree ./includes
```

Comment:
As I was inspecting our generated CSS in chrome inspector, I noticed something really odd. `application.tailwind.css` which lives in the /assets/stylesheets/ directory was being compiled into tailwind.css (good) but it was *also* being included raw in it's uncompiled state. I fixed it by updating this line.

The implication of this /includes change is that any additional CSS files we want to create, over time, need to be put within the /includes directory. They cannot be put directly in the stylesheets/ directory.

What's odd is that all of this is plain vanilla generated rails config, so it may be a bug with rails 7.

I considered:

1. Moving application.tailwind.css to another directory, but tailwindcss is configured to look for it where it is
2. Removing require_tree altogether and just manually requiring any future CSS files we put in stylesheets/ but that feels like it would be more annoying.

Best solution I could come up with was to kill the `require_tree .` in favor of `require_tree ./includes'

Reply:
I think that's fine. With tailwind I rarely ever need to declare styles manually. You tend to do them once or twice and then you're done. 

Actually, given how infrequent this is likely to be, it might be worth putting either a comment here in code explaining this, or moving the few declarations we have for tooltips into the includes directory to set an example to follow.

Reply:
Good idea! I'll pull them out to start the pattern.